ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -619,8 +751,49 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: baremetal-lab-upi
-- as: gcp-ipi-oidc-rt-fips-amd-f14
-  cron: 25 18 1,17 * *
+- as: gcp-ipi-basecap-none-additionalcaps-capi-arm-f28
+  cron: 15 23 9 * *
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      BASELINE_CAPABILITY_SET: None
+      COMPUTE_NODE_TYPE: t2a-standard-4
+      FEATURE_GATES: '["ClusterAPIInstall=true"]'
+      FEATURE_SET: CustomNoUpgrade
+      OCP_ARCH: arm64
+      PERSISTENT_MONITORING: "false"
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-gcp-ipi-capability-additionalcaps
+- as: gcp-ipi-compact-filestore-csi-capi-arm-f28
+  cron: 30 22 14 * *
+  steps:
+    cluster_profile: gcp-qe
+    dependency_overrides:
+      OO_INDEX: quay.io/openshift-qe-optional-operators/aosqe-index:v4.16
+    env:
+      COMPUTE_NODE_TYPE: t2a-standard-4
+      FEATURE_GATES: '["ClusterAPIInstall=true"]'
+      FEATURE_SET: CustomNoUpgrade
+      OCP_ARCH: arm64
+      SIZE_VARIANT: compact
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-gcp-ipi-filestore-csi
+- as: gcp-ipi-disk-encryption-capi-arm-f28
+  cron: 45 23 18 * *
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      COMPUTE_NODE_TYPE: t2a-standard-4
+      FEATURE_GATES: '["ClusterAPIInstall=true"]'
+      FEATURE_SET: CustomNoUpgrade
```

Comment:
I have concern this job would can not be upgraded, the `CustomNoUpgrade` feature set is introduced for testing `capi` which is from a installer feature, for upgrade, we can run the upgrade for the profile by removing `capi` keyword, `FEATURE_GATES` and `FEATURE_GATES` setting.

Reply:
Is capi a TP feature? If true, should we add tp in the profile name?

Reply:
@jianli-wei @jinyunma @sunzhaohua2 Could you please take a look at following jobs, do they support upgrade?
```
- as: gcp-ipi-basecap-none-additionalcaps-capi-arm-f7
  cron: 41 17 8,24 * *
  steps:
    cluster_profile: gcp-qe
    dependencies:
      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
    env:
      BASELINE_CAPABILITY_SET: None
      COMPUTE_NODE_TYPE: t2a-standard-4
      FEATURE_GATES: '["ClusterAPIInstall=true"]'
      FEATURE_SET: CustomNoUpgrade
      OCP_ARCH: arm64
      PERSISTENT_MONITORING: "false"
    test:
    - chain: openshift-e2e-test-qe
    workflow: cucushift-installer-rehearse-gcp-ipi-capability-additionalcaps
- as: gcp-ipi-compact-filestore-csi-capi-arm-f14
  cron: 2 1 9,25 * *
  steps:
    cluster_profile: gcp-qe
    dependencies:
      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
    dependency_overrides:
      OO_INDEX: quay.io/openshift-qe-optional-operators/aosqe-index:v4.16
    env:
      COMPUTE_NODE_TYPE: t2a-standard-4
      E2E_RUN_TAGS: not @storage
      FEATURE_GATES: '["ClusterAPIInstall=true"]'
      FEATURE_SET: CustomNoUpgrade
      OCP_ARCH: arm64
      SIZE_VARIANT: compact
    test:
    - chain: openshift-e2e-test-qe
    workflow: cucushift-installer-rehearse-gcp-ipi-filestore-csi
- as: gcp-ipi-disk-encryption-capi-arm-f14
  cron: 1 19 6,20 * *
  steps:
    cluster_profile: gcp-qe
    dependencies:
      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
    env:
      COMPUTE_NODE_TYPE: t2a-standard-4
      FEATURE_GATES: '["ClusterAPIInstall=true"]'
      FEATURE_SET: CustomNoUpgrade
      OCP_ARCH: arm64
    test:
    - chain: openshift-e2e-test-qe
    workflow: cucushift-installer-rehearse-gcp-ipi-disk-encryption
- as: gcp-ipi-ovn-ipsec-capi-arm-mixarch-f14
  cron: 28 0 15,29 * *
  steps:
    cluster_profile: gcp-qe
    dependencies:
      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
    env:
      ADDITIONAL_WORKER_ARCHITECTURE: x86_64
      ADDITIONAL_WORKER_VM_TYPE: n2-standard-4
      ADDITIONAL_WORKERS: "1"
      COMPUTE_NODE_REPLICAS: "2"
      COMPUTE_NODE_TYPE: t2a-standard-4
      FEATURE_GATES: '["ClusterAPIInstall=true"]'
      FEATURE_SET: CustomNoUpgrade
      OCP_ARCH: arm64
    test:
    - ref: ipi-install-heterogeneous
    - chain: openshift-e2e-test-qe
    workflow: cucushift-installer-rehearse-gcp-ipi-ovn-ipsec
```

Reply:
> Is capi a TP feature? If true, should we add tp in the profile name?

CAPI is being heavily developed and tested, to avoid affect the normal testing, we can enabled `["ClusterAPIInstall=true"]` FEATURE_GATES + CustomNoUpgrade to only enabled API. And if also can enable the TP for the whole clutser via `FEATURE_SET: TechPreviewNoUpgrade` to run the CAPI, but that would also enable all TP features, not only CAPI. So we do not have to plan to add `tp` in profile name.

As our plan, we are going to GA CAPI on azure and gcp in 4.17 release cycle.

Reply:
Got it. Thank you!
@JianLi-RH Before CAPI goes to GA, I think it only supports z-stream upgrade.